src/applications/simple-forms/form-upload/containers/ConfirmationPage.jsx

```diff
@@ -64,9 +64,26 @@ const ConfirmationPage = () => {
           text="Print this page"
         />
       </div>
-      <a className="vads-c-action-link--green vads-u-margin-y--2" href="/">
-        Go back to VA.gov
-      </a>
+      <div
+        className="action-bar-arrow vads-u-margin-bottom--4"
+        style={{ maxWidth: '60%' }}
```

Comment:
curious how these look on mobile. you can change to mobile view in dev tools to try it out.

Reply:
Good catch! It looked pretty bad. Raising this to `70%` helped a lot.